packages/tasks/src/tasks/index.ts

```diff
@@ -216,6 +216,7 @@ export interface TaskData {
 	datasets: ExampleRepo[];
 	demo: TaskDemo;
 	id: PipelineType;
+	canonicalId?: PipelineType;
```

Comment:
Added this property to express one task being a "subtask" of another (eg, summarization being a subtask of text2text-generation)